include/DoocsBackendImageRegisterAccessor.h

```diff
@@ -0,0 +1,142 @@
+// SPDX-FileCopyrightText: Deutsches Elektronen-Synchrotron DESY, MSK, ChimeraTK Project <chimeratk-support@desy.de>
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#pragma once
+
+#include "DoocsBackendRegisterAccessor.h"
+#include <eq_client.h>
+#include <type_traits>
+
+#include <ChimeraTK/MappedImage.h>
+
+namespace ChimeraTK {
+
+  /// implements conversion from DOOCS image -> MappedImage
+  struct MappedDoocsImgIn : public MappedImage {
+    using MappedImage::MappedImage;
+
+    ImgFormat getImgFormat(const IMH* h) {
```

Comment:
Please move implementation to .cc file. (If not possible e.g. for templates, move them at least below the class declaration, like the inline functions below - btw: inlining is no performance here, so please move them to the .cc file as well)

Reply:
done